apps/studio/components/interfaces/GraphQL/GraphiQL.tsx

```diff
@@ -354,14 +381,29 @@ export const GraphiQLInterface = ({ theme }: GraphiQLInterfaceProps) => {
               <div ref={editorResize.secondRef}>
                 <div
                   className={clsx(
-                    'graphiql-response text-sm',
+                    'graphiql-response text-sm relative',
                     hasSingleTab
                       ? styles.graphiqlResponseSingleTab
                       : styles.graphiqlResponseMultiTab
                   )}
                 >
                   {executionContext.isFetching ? <Spinner /> : null}
                   <ResponseEditor />
+
+                  <Alert_Shadcn_ variant="warning" className="absolute bottom-[5px] right-[5px]">
+                    <IconAlertTriangle strokeWidth={2} />
+                    <AlertTitle_Shadcn_ className="leading-5">
+                      Please note that queries and mutations run in GraphiQL now use the service
+                      role key by default.
+                      <br />
```

Comment:
do we need the line break here? looks a bit weird
![image](https://github.com/supabase/supabase/assets/19742402/98d4b0c7-d6b9-4edf-a7e3-ed4692881209)

I'm just thinking if we might be able to spin this into a positive, rather than a warning, so perhaps something like
"Run queries and mutations as a specific role/user via role impersonation"
"GraphiQL will use the service role key by default and will bypass any RLS policies"

cc @saltcod wondering what you think here?

but not a blocker!

Reply:
also wondering if this should be dismissable? devs might want the real estate to view their results

Reply:
side note - those text colors are weird

Reply:
It's now dismissible and using `text-foreground` on the title. Terry can play with the language as he sees fit ;)